DOCS-5008 Fixed typo (#336)
mulesoft/docs-connectors: modules/ROOT/pages/xml/xml-xslt.adoc

The `<xml-module:xslt-transform>` element can take an input XML document and transform it using an XSL style sheet.

> MuleSoft supports the XSLT standard, but DataWeave is capable of achieving the same use cases and is the recommended tool for extracting and transforming XML documents.

Reviewer: Another way of saying this that puts the "but" first is "Although MuleSoft supports, DataWeave can do it too."
You might want that emphasis; or, you might not! Your decision; both are fine.

Reviewer: is capable of > can

> This is an example input XML document:

Reviewer: Forgive my ignorance: can you review and be sure that all these modifiers are in the right position:

     example > input > XML > document?

Is it a document serving as an example of XML input? Is it an XML document that shows example input? Or some other combination? I could not parse the string with any certainty, but your audience might be just fine with it.

Reviewer: Ah! I see below that you use "the XML input document," so I think what you want is "example XML input document" above.

</xml-module:xslt-transform>
----

> The script above outputs the following document:

Reviewer: The script above > This script

</table>
----

> Although by default this operation looks for the input document at the message payload level, you can supply your own input. For example, imagine that the cities were a JSON array inside the payload. You could first transform the JSON to XML, and then run the XSL transformation. For example:

Reviewer: ```suggestion
Although this operation looks for the input document at the message payload level by default, you can supply your own input. For example, imagine that the cities are a JSON array inside the payload. You could then first transform the JSON to XML, and then run the XSL transformation:
```

</flow>
----

. The example flow is triggered through an HTTP request.

> . The example assumes that the `userId` is provided as a query parameter and uses the File connector to read the file that contains the correct XSL and stores it in a variable.

Reviewer: ```suggestion
. The example is based on the `userId` value being provided as a query parameter and the File connector being used to read the file that contains the correct XSL and store it in a variable.
```

Reviewer: Is File the name of the connector, or is it a type of connector in general?
If the former then...the short official name is File Connector (?) 
If the latter, lowercase: the file connector

. The example then executes a transformation using an expression to point to the XSL sheet obtained through the `Read` operation.

> The HTTP and File connectors are used as examples and are not required to perform similar use cases.

Reviewer: ditto re. connector name vs. type of connector

. The XSL style sheet declares a `param` called `vtn`.
. The style sheet references the assigned value of the `param` using the `$` prefix.

> . The example uses the `context-properties` parameter to pass the values in.

Reviewer: "to pass the values in" > "to pass in the values" or "to import the values"

(The verb is "to pass in.")

== Output MIME Type

> The most common use case for XSLT is to output a new XML document. However, XSLT can produce documents in other formats, such as JSON, CSV, or a text or number value, which means that the produced value will not have any specific MIME type assigned. You should specify a MIME type using the `outputMimeType` parameter.

Reviewer: ```suggestion
The most common use case for XSLT is to output a new XML document. However, XSLT can produce documents in other formats, such as JSON, CSV, or a text or number value, which means that the produced value does not have any specific MIME type assigned. You should specify a MIME type using the `outputMimeType` parameter.
```
